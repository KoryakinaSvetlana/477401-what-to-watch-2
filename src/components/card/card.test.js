import React from "react";
import renderer from "react-test-renderer";
import Card from "./card.js";

it(`App correctly renders after relaunch`, () => {
  const tree = renderer.create(
      <Card
        film={`1`}
        onHeaderClick={() => {}}
      />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
