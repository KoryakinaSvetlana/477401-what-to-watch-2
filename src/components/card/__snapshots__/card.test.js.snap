// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App correctly renders after relaunch 1`] = `
<article
  className="small-movie-card catalog__movies-card"
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="1"
      height="175"
      src="https://via.placeholder.com/280x175"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      1
    </a>
  </h3>
</article>
`;
